setup.py

```diff
@@ -23,7 +23,7 @@
 
 install_requires = [
     'six>=1.7.2',
-    'isbnlib>=3.5.6',
+    'isbnid==0.3.4',
```

Comment:
See:
* [build](https://travis-ci.org/inveniosoftware/idutils/builds/138048297) + [commit](https://github.com/inveniosoftware/idutils/commit/27c5f2adf08c694a3ff741ff101bcf38874b2e75) for `0.4.2.1`
* [build](https://travis-ci.org/inveniosoftware/idutils/builds/138015686) + [commit](https://github.com/inveniosoftware/idutils/commit/669c30f8d55e7c81136124c5182577988c4f8ee6) for `>=0.4.2`

Maybe >= would also work, but given the weird release numbering it might be more stable to just pin to a version that works and forget about it :)